Slam/SlamAPI.swift

```diff
@@ -1,16 +1,32 @@
 import Foundation
 
 public class SlamAPI {
-    public let getCurrentQueueURL = "http://slamapi.herokuapp.com/matches"
+    public let getMatchesURL = "http://slamapi.herokuapp.com/matches"
     let httpClient: HTTPClient
 
     public init(httpClient: HTTPClient) {
         self.httpClient = httpClient
     }
 
+    public func addMatch(playerOne: String, playerTwo: String) {
+        var request = NSMutableURLRequest(
+            URL: NSURL(string: getMatchesURL)!
```

Comment:
`getMatchesURL` is a weird name for an endpoint that is used to "POST" data.

Even though they currently have the same value, I would make a separate constant for this endpoint. There's nothing enforcing it to remain the same.